scripts/release.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+set -e
+
+IS_PRERELEASE=true
+
+if [[ ! $- == *i* ]]
+then
+    echo "Warning: this is an interactive script"
+fi
+
+if [[ -z $CIRCLE_CI_TOKEN ]]
+then
+  echo "Please provide a CircleCI API token in \$CIRCLE_CI_TOKEN"
+  exit 1
+fi
+
+if [[ -z $GITHUB_TOKEN ]]
+then
+  echo "Please provide a Github personal access token in \$GITHUB_TOKEN"
+  exit 1
+fi
+
+if [[ -z $GOPATH ]]
+then
+  echo "Please install Golang"
+  exit 1
+fi
+
+if [[ $(ls $GOPATH/bin/hub | wc -l) == "0" ]]
+then
+  echo "Installing required tool Github 'hub'"
+  go get github.com/github/hub
+fi
+
+read -p "Creating a release off \"$(git branch | grep \* | cut -d ' ' -f2)\", is this correct? [Y/n] "  Y
+if [[ -n $Y && "$(echo $Y | tr '[:upper:]' '[:lower:]')" != "y" ]]
+then
+  exit 0
+fi
+
+read -p "Enter release version (eg v1.2.3 or test-myfeature) " VERSION
```

Comment:
Might want to check the value is sensible.
(Not empty, looks more or less like a valid tag)

Reply:
If it's empty or already used, git tag fails